test/podFailover/controller/main.go

```diff
@@ -155,10 +157,10 @@ func createTestNamespace(ctx context.Context, clientset *kubernetes.Clientset) e
 		}
 		return false, nil
 	}); err != nil {
-		return err
+		return true, err
 	}
 
-	return nil
+	return true, nil
```

Comment:
I don't see any cases where we would return `false, nil`, so why the addition of the `bool` return when `err != nil` should suffice to prevent an attempt to delete the namespace?

Reply:
Added

Reply:
I missed adding the case where we return false, nil. Added that now.

Reply:
Ah, this makes more sense now.